website/content/docs/agent/wal-logstore/revert-to-boltdb.mdx

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+page_title: Revert to BoltDB
+description: >-
+  Learn how to safely configure and test the experimental WAL backend in your Consul deployment.
+---
+
+# Revert storage backend to BoltDB from WAL 
+
+This topic describes revert your Consul storage backend from the experimental WAL LogStorage backend to the default BoltDB.
+
+The overall process for reverting to BoltDB consists of the following steps. Repeat the steps for all Consul servers that you need to revrt.
+
+1. Select target server.
+1. Stop target server gracefully.
+1. Remove data directory from target server.
+1. Update target server's configuration.
+1. Start target server.
+
+## Select target server
+
+If you are using Consul OSS or Consul Enterprise without non-voting servers, select one of the follower servers. As noted in [Risks](/consul/docs/agent/wal-logstore/enable#risks), Consul Enterprise users with non-voting servers should begin the procedure with a non-voting server.
+
+Run the following command to retrieve the current state of the servers:
+
+```shell-session
+$ consul operator raft list-peers
+```
```

Comment:
This doesn't seem relevant for this guide - the target server is just whichever one is running WAL that you need to revert. Not sure if this needs a step?